### ISSUE TITLE
Experiment: CDS YOLO run

### ISSUE THREAD

**Issue by gonzalobenegas** [Open-Athena/marin-dna#27]
## Description

Train a model of a modest size (~1B) with reasonable defaults on an animal CDS (coding sequence) dataset.

## Hypothesis or Goal

See how far you can get in downstream tasks and understand major strengths and weaknesses to guide future experiments.

### Links

[Training code](https://github.com/marin-community/marin/blob/bc69e0fbce91fd2e2a30bbcda1b9a4f43fbcd858/experiments/dna/exp27_cds_yolo.py)  
[Analysis code](https://github.com/Open-Athena/bolinas-dna/tree/main/snakemake/analysis/evals_v1)  
[Wandb](https://wandb.ai/gonzalobenegas/marin/runs/animal-cds-yolo-repeat-weight-0.01-r01-e8cb9b?nw=nwusergonzalobenegas)

### Variant effect prediction

Setup:
- Classification of human causal missense variants from ClinVar (mostly Mendelian) and UK BioBank (complex traits), as defined in [GPN-Star paper](https://www.biorxiv.org/content/10.1101/2025.09.21.677619v1)
- Odd chromosomes only (validation set)
- ClinVar subset to 5000 positives and negatives

Results:
- Matches Evo 2, behind GPN-Star (similar patterns to https://github.com/Open-Athena/bolinas-dna/issues/21)

<img width="1501" height="755" alt="Image" src="https://github.com/user-attachments/assets/b0066daa-7463-4b3a-9d88-694332bd70ad" />

<img width="300" height="300" alt="Image" src="https://github.com/user-attachments/assets/e46b402d-8ac8-4002-a067-cf79ad62f31e" />

**Comment by gonzalobenegas** [comment]
🤖 Updated plot using the post-#194 evals_v2 dataset (`mendelian_traits`, hf_revision `4aed58e`).

![exp27_cds_yolo_auprc](https://gist.githubusercontent.com/gonzalobenegas/3649e68fb63ca1f3443e4486078eb4d8/raw/c785daa5a1a938365ace6861791fa3e978ac53cf/exp27_cds_yolo_auprc.svg)

Recipe: [`plots/exp27_cds_yolo.py`](https://github.com/Open-Athena/marin-dna/blob/0058ec4779401307a9b3a3c9218127511711bce4/plots/exp27_cds_yolo.py)
